### ISSUE TITLE
node fibres

### ISSUE THREAD

**Issue by maxkfranz** [PathwayCommons/factoid#1243]
We could remove node fibres with worker threads so we don't have so many node compatibility issues in future. Worker threads didn't exist when we started this project.

- Ref: https://nodejs.org/api/worker_threads.html

**Comment by jvwong** [comment]
Is this related to [`fibers`](https://www.npmjs.com/package/fibers)? This repo currently version works with v16.  

The one I am concerned about is [grounding-search](https://github.com/PathwayCommons/grounding-search) which I suspect will be more actively developed.

**Comment by maxkfranz** [comment]
> Is this related to [fibers](https://www.npmjs.com/package/fibers)?

Yeah, I just went with the tab I already had open.  I wanted to track this quickly while going through emails.

**Comment by jvwong** [comment]
OK I'll post it there too.
